File: BOJ/Math/BJ10250.cpp
#include <iostream>
#include <string>
using namespace std;

int T, H, W, N;

string under_ten(int num) {
  string zero = "0";
  return num >= 10 ? to_string(num) : zero.append(to_string(num));
}

int main() {
  ios::sync_with_stdio(0);
  cin.tie(0);

  cin >> T;
  for(int t=0; t<T; t++) {
    cin >> H >> W >> N;

    int floor = N % H;
    int room = N / H;

    string res;
    if(floor) res = to_string(floor) + under_ten(room + 1);
    else res = to_string(H) + under_ten(room);

    cout << res << '\n';
  }
  return 0;
}

File: BOJ/Math/BJ10250.js
const fs = require('fs');
const [T, ...args] = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

function underTen(num) {
  return num < 10 ? '0' + num : num.toString();
}

const arr = args.reduce((acc, cur) => {
  const curArr = cur.split(' ').map(Number);
  acc.push(curArr);
  return acc;
}, []);

arr.forEach((info, i) => {
  const [H, W, N] = info;
  const floor = N % H;
  const room = Math.floor(N / H);

  if(floor) console.log((floor).toString().concat(underTen(room+1)));
  else console.log(H.toString().concat(underTen(room)));
});